theme/brb/views/lib/footer.hbs

```diff
@@ -1,25 +1,37 @@
-<footer class="hide-inline small-footer" id="footer">
+<footer id="footer" class="hide-inline small-footer" tabindex="0">
 	<div class="imprint">
 		<div class="center">
-			<img src="/images/footer-logo.png" alt="Anbieterlogo" />
+			<img src="/images/footer-logo.png" alt="" role="presentation"/>
```

Comment:
why did you remove the alt here?

Reply:
This is not a link and looked like it is only for presentational purposes?

Reply:
👍